index.html

```diff
@@ -6041,7 +6041,7 @@ <h4 id="font_selection_and_font_size_of_heading">
             </ol>
           </aside>
           <aside class="note" id="n206">
-            <p its-locale-filter-list="en" lang="en">Small-headings are sometimes set with a Japanese gothic face and smaller character size than the character size of the main text. For example, for 8 point mincho main text, set the small-headings with 8 point Japanese gothic.</p>
+            <p its-locale-filter-list="en" lang="en">Low level headings are sometimes set with a Japanese gothic face and smaller character size than the character size of the main text. For example, for 8 point mincho main text, set the low level headings with 8 point Japanese gothic.</p>
```

Comment:
As we are making heading names consistent, i.e. top, medium and small, I am not sure if this change makes sense. There are a few other similar changes.

Reply:
https://github.com/w3c/jlreq/issues/120#issuecomment-642367558 says top/medium/low... (and pending low or small?)

Reply:
oh, you are right. It is top, medium and low. Please ignore my comment.